### ISSUE TITLE
swap team names in debate setup page

### ISSUE THREAD

**Issue by jakubmanczak** [debatecore/debate-tools#31]
Add an option to swap the text content of the team names.

<img width="672" alt="image" src="https://github.com/debatecore/debate-tools/assets/56172798/6ffc273d-38f2-4cd3-9f37-f34f8b1091dd">
<br />
<img width="133" alt="image" src="https://github.com/debatecore/debate-tools/assets/56172798/978fb0ae-5315-47b6-9260-f33c9d5973e3">

e.g. GenericButton with some kind of "swap" icon in between the inputs?

**Comment by Mateusz-Dobrzynski** [comment]
This one should be implemented alongside #27 so that the setup view matches the debate conduction view. Clicking the swap button should change both views.

**Comment by jakubmanczak** [comment]
> Clicking the swap button should change both views.

I don't think it should? The functionalities seem to be very similar, but are very much distinct.

**Comment by Mateusz-Dobrzynski** [comment]
I've re-read this issue and now I can see that I misunderstood the intended feature at first. Yes, they are very much distinct. #27 will invert the physical placement of the teams relative to the clock, whereas this issue will change the sides of the debate (i.e. if ZSK is the proposition and 38 LO is the opposition, clicking the swap button will put ZSK in the opposition and 38 LO in the proposition).

We need both features, but they must be distinguishable because both swap buttons will be in the config screen (or the config wizard). When making the wizard, we could create an interactive debate schema that appears in many introductory materials.

![oxford debate schema](https://github.com/debatecore/debate-tools/assets/60257096/2d4b175b-88c9-492c-95a1-1d64bf75eabc)
~Based on the schema used in a Polish [Oxford debate article](https://commons.wikimedia.org/wiki/File:Debata_oksfordzka_-_uk%C5%82ad_przestrzenny.jpg) on Wikipedia.

One button would swap the proposition name with the opposition name. The _Proposition_ and _Opposition_ captions would stay the same to provide a frame of reference, just like the judges' marshall's tables. The other button would move the clock from the top of the schema to the bottom. How about that?
